packages/app/package.json

```diff
@@ -14,6 +14,10 @@
     "react-native-reanimated": "3.0.2"
   },
   "peerDependencies": {
+    "@metaplex-foundation/js": "^0.18.1",
+    "@mysten/sui.js": "0.30.0",
+    "@solana/spl-token": "^0.3.7",
+    "@solana/web3.js": "1.75.0",
```

Comment:
since there was deps version update in between.. use these version:
```json
{
    "@metaplex-foundation/js": "0.18.3",
    "@mysten/sui.js": "0.32.1",
    "@solana/spl-token": "0.3.7",
    "@solana/web3.js": "1.75.0",
}
```